ruby/vampires.rb

```diff
@@ -0,0 +1,53 @@
+puts "How many employees are you processing today?"
+  x = gets.chomp.to_i
+
+while x > 0
+
+  puts "What is your name?"
+  name = gets.chomp
+  puts "How old are you?"
+  age = gets.chomp.to_i
+  puts "What year were you born in?"
+  year = gets.chomp.to_i
+  puts "Our company cafeteria orders garlic bread...should we order some for you? (yes or no)"
+  garlic_bread = gets.chomp
+  puts "Would you like to enroll in our company's health insurance?"
+  health_insurance = gets.chomp
+
+age_match = 2017 - year
+
+if (name != "Drake Cula" && name != "Tu Tang") && (age == age_match) && (garlic_bread == "yes" || health_insurance == "yes")
```

Comment:
These results should be at the bottom of the file.

When using a control flow it is important to complete the evaluation before returning any results.  I see that you finishing with the allergy check but it would be a better practice to get the allergy information during the rest of the interview and make your evaluation after the employee answered all of the questions.

Give this a quick refactor and let me know if you have any questions.